doc/operator-monitoring-resources.md

```diff
@@ -0,0 +1,51 @@
+# 3scale Monitoring Resources
+
+The 3scale monitoring resources get installed when 3scale is installed on Openshift using the 3scale Operator.
```

Comment:
Should we clarify that is only installed when the appropriate setting is enabled?

Reply:
I see it's specified a little bit below in `Enabling 3scale monitoring`

Reply:
fixed